src/allFiles.ts

```diff
@@ -0,0 +1,6 @@
+import { walk } from "./promisified"
```

Comment:
you already have a file `fileUtils` doesn't this function belong to it?

Reply:
Moved to `fileUtils.ts` ✅